#
# old_revision [3b3436144d2e1465d09bf786689280c39eaaa3ef]
#
# add_file "xarinit.php"
#  content [5b2c76c79b83621e60c277f78264b8f2ae60c556]
#
============================================================
--- xarinit.php	5b2c76c79b83621e60c277f78264b8f2ae60c556
+++ xarinit.php	5b2c76c79b83621e60c277f78264b8f2ae60c556
@@ -0,0 +1,40 @@
+<?php
+/**
+ * File: $Id$
+ *
+ * SimplePie init function. Xaraya 1.1.5 seems to need this.
+ *
+ * @package Xaraya eXtensible Management System
+ * @copyright (C) 2006 by the Xaraya Development Team.
+ * @license GPL {@link http://www.gnu.org/licenses/gpl.html}
+ * @link http://www.xaraya.com
+ *
+ * @subpackage jobs
+ * @author Jason Judge
+ */
+function simplepie_init()
+{
+
+    // Initialisation successful
+    return true;
+}
+function simplepie_upgrade($oldversion)
+{
+    switch($oldversion){
+       case '0.0.1':
+         
+        break;
+
+    }
+    return true;
+}
+/**
+ * delete the smiley module
+ */
+function simplepie_delete()
+{
+
+    // Deletion successful
+    return true;
+}
+?>
